third_party/BUILD.buck

```diff
@@ -6,9 +13,209 @@ cxx_library(
+        "cpuinfo/wrappers/linux/smallfile.c",
+        "cpuinfo/wrappers/mach/topology.c",
+        "cpuinfo/wrappers/x86/cache/descriptor.c",
+        "cpuinfo/wrappers/x86/cache/deterministic.c",
+        "cpuinfo/wrappers/x86/cache/init.c",
+        "cpuinfo/wrappers/x86/info.c",
+        "cpuinfo/wrappers/x86/init.c",
+        "cpuinfo/wrappers/x86/isa.c",
+        "cpuinfo/wrappers/x86/linux/cpuinfo.c",
+        "cpuinfo/wrappers/x86/linux/init.c",
+        "cpuinfo/wrappers/x86/mach/init.c",
+        "cpuinfo/wrappers/x86/name.c",
+        "cpuinfo/wrappers/x86/topology.c",
+        "cpuinfo/wrappers/x86/uarch.c",
+        "cpuinfo/wrappers/x86/vendor.c",
+        "cpuinfo/wrappers/x86/windows/init.c",
+    ],
+    include_directories = ["cpuinfo/src"],
+    public_include_directories = ["cpuinfo/include"],
+    raw_headers = glob([
+        "cpuinfo/src/**/*.h",
+        "cpuinfo/src/**/*.c",
+    ]),
+    preferred_linkage = "static",
+    preprocessor_flags = [
+        "-DCPUINFO_LOG_LEVEL=2",
+        "-D_GNU_SOURCE=1",
+    ],
+    visibility = ["PUBLIC"],
+    deps = [
+        ":clog",
+    ],
+)
+
+cxx_library(
+    name = "clog",
+    srcs = [
+        "cpuinfo/deps/clog/src/clog.c",
+    ],
+    raw_headers = glob([
+        "cpuinfo/deps/clog/include/*.h",
+    ]),
+    public_include_directories = [
+        "cpuinfo/deps/clog/include/",
+    ],
+    force_static = True,
+    visibility = ["PUBLIC"],
+)
+
+cxx_library(
+    name = "FP16",
+    raw_headers = glob([
+        "FP16/include/*.h",
+    ]),
+    public_include_directories = [
+        "FP16/include/",
+    ],
+    force_static = True,
+    visibility = ["PUBLIC"],
+)
+
+
+cxx_library(
+    name = "miniz",
+    srcs = ["miniz-2.0.8/miniz.c"],
+    header_namespace = "",
+    exported_headers = {"miniz.h": "miniz-2.0.8/miniz.h"},
+    exported_preprocessor_flags = [
+        "-DMINIZ_DISABLE_ZIP_READER_CRC32_CHECKS",
+    ],
+    visibility = ["PUBLIC"],
+)
+
+remote_file(
+  name = "typing-extensions-download",
+  url = "https://files.pythonhosted.org/packages/75/e1/932e06004039dd670c9d5e1df0cd606bf46e29a28e65d5bb28e894ea29c9/typing_extensions-4.2.0-py3-none-any.whl",
+  sha1 = "ff0849420e94f425818bff5d0f25e3cdfaba8601",
+  out = "typing_extensions-4.2.0-py3-none-any.whl",
+)
+
+prebuilt_python_library(
+  name = "typing-extensions",
+  binary_src = ":typing-extensions-download",
+  visibility = ["PUBLIC"],
+  deps = [":typing-extensions-download"],
+)
+
+remote_file(
+  name = "pyyaml-download",
+  url = "https://files.pythonhosted.org/packages/12/fc/a4d5a7554e0067677823f7265cb3ae22aed8a238560b5133b58cda252dad/PyYAML-6.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_12_x86_64.manylinux2010_x86_64.whl",
+  sha1 = "11aa9c5fe2d890b6a73212beadc7c8a4265ebc39",
+  out = "pyyaml.whl",
+)
+
+prebuilt_python_library(
+  name = "pyyaml",
+  binary_src = ":pyyaml-download",
+  visibility = ["PUBLIC"],
+  deps = [":pyyaml-download"],
+)
```

Comment:
Why do we need these?  Aren't these already installed in the CI containers?

Reply:
I add them because it didn't work on my local machine (I already installed them using conda). Maybe just my problem. Let me check if CI can work without them.

Reply:
I don't know why but it's not working. 
  File "torchgen/gen.py", line 3, in <module>
ModuleNotFoundError: No module named 'typing_extensions'

Reply:
BUCK cannot see system installed packages. I made a post in BUCK Q&A but haven't found a solution. I will checkin these rules for now.